src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+const CHECK_STATUS = 'CHECK_STATUS';
+
+const categories = [];
+
+export const CheckStatusFunc = () => ({
+  type: CHECK_STATUS,
+});
+
+const CategoriesReducer = (state = categories, action) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return ['Under construction'];
+    default: return state;
+  }
+};
```

Comment:
As previously mentioned, you are supposed to refactor the logic here to use `createSlice` just as you have done with the books.js. Please do this to keep up with the modern approach and to stick with the milestone's requirement.